interchaintest/docker.go

```diff
@@ -29,7 +30,15 @@ type dockerErrorDetail struct {
 	Message string `json:"message"`
 }
 
-func BuildRelayerImage(t *testing.T) {
+func UniqueRelayerImageName() string {
+	uuid, err := uuid.NewRandom()
+	if err != nil {
+		fmt.Printf("Failed to generate UUID: %v\n", err)
+	}
+	return RelayerImagePrefix + uuid.String()[:6]
+}
+
+func BuildRelayerImage(t *testing.T, uniquestr string) {
```

Comment:
nit: rename `uniquestr` to something that more clearly defines what this variable is, e.g. `image`